SimG4CMS/Calo/interface/CaloSD.h

```diff
@@ -49,23 +49,23 @@ class CaloSD : public SensitiveCaloDetector,
 
 public:    
   
-  CaloSD(G4String  aSDname, const DDCompactView & cpv,
+  CaloSD(std::string aSDname, const DDCompactView & cpv,
```

Comment:
Hi David, it is possible to use const std::string in CMSSW classes but the change requires addition of similar fix to 12 more files. It is done and I trigger tests now.